tracing-attributes/tests/ui/async_instrument.stderr

```diff
@@ -93,6 +93,6 @@ error[E0308]: mismatched types
 42 |   async fn extra_semicolon() -> i32 {
    |  ___________________________________^
 43 | |     1;
-   | |      - help: remove this semicolon
+   | |      - help: remove this semicolon to return this value
```

Comment:
This change is intended to fix the `cargo test` error in CI: https://github.com/tokio-rs/tracing/actions/runs/4646473012/jobs/8222940082 .

If it should be in a separate PR, please let me know.